src/util.cpp

```diff
@@ -1886,7 +1890,15 @@ QString GetVersionAndNameStr ( const bool bWithHtml )
         strVersionText += "\n *** ";
     }
 
-    strVersionText += QCoreApplication::tr ( "Released under the GNU General Public License (GPL)" );
+    strVersionText += QCoreApplication::tr ( "Released under the GNU General Public License version 2 (GPLv2)" );
+
+    if ( !bWithHtml )
```

Comment:
Shouldn't `bWithHtml` only affect formatting, not content?

Reply:
I have used this, because it did not make to me to have the Qt details shown in the part of the GUI about dialog.
Good point about bWithHtml being more just about formatting.

Any objections in including the Qt reference in the GUI here as well?